### PR TITLE
[UPDATE] Check Http Service

### PR DESCRIPTION
**Pull request by ramonornela**
- Support array to HttpServiceCheck ticket https://github.com/liip/LiipMonitorBundle/issues/32

### CODE REVIEW

**Comment by ramonornela** [comment]
ping @lsmith77

File: Check/HttpServiceCheck.php [hunk 1]
```diff
@@ -5,76 +5,155 @@
...
 {
     /**
      * @var string
      */
-    protected $host;
+    protected $hosts;
 
     /**
-     * @var int
+     * @param array $host
```
Comment (lsmith77) [inline Check/HttpServiceCheck.php:18]: this doesnt seem correct

File: Check/HttpServiceCheck.php [hunk 2]
```diff
@@ -5,76 +5,155 @@
...
+        }
+
+        return $this;
+    }
 
     /**
-     * @var int
+     * @param string $alias
+     * @param array  $options
+     * @return       $this
```
Comment (lsmith77) [inline Check/HttpServiceCheck.php:117]: not sure what the convention is for fluent interfaces, but I would expect a type to be specified

**Comment by lsmith77** [comment]
i know we currently have no tests but this change significantly complicates the code, so it would be great if you could add a test

**Pull request closed without merging**